public/registration.html

```diff
@@ -49,7 +49,7 @@ <h2>Registrace výsledků</h2>
                 <label for="date">Datum:</label>
             </div>
             <div class="col-75">
-                <input type="date" id="date" name="date" data-bind="value: date, event: { input: onDateChange }">
+                <input type="date" id="date" name="date" data-bind="value: date, event: { input: onDateChange }, attr: { min: '2021-01-10', max: moment().format('YYYY-MM-DD') }">
```

Comment:
so is it moment() or moment(new Date())

Reply:
It is both the same, parameter is optional, current date and time are used if it is missing. https://momentjs.com/